client/post-editor/media-modal/secondary-actions.jsx

```diff
@@ -32,14 +33,13 @@ const MediaModalSecondaryActions = React.createClass( {
 		activeView: React.PropTypes.oneOf( values( ModalViews ) ),
 		disabled: PropTypes.bool,
 		onDelete: PropTypes.func,
-		onChangeView: PropTypes.func
```

Comment:
>If we passed here `onEdit` instead of 'setView', it would be easier to use this outside of a context of this specific modal

This is a good point, and is relevant to our discussion yesterday. I renamed many of the props in 881ceb7 to help with this distinction.

I'm not sold one way or the other on the "on" prefix, but I used them here anyways.

cc also @dmsnell, includes analytics usage per our recent discussion